src/js/player.js

```diff
@@ -2218,27 +2219,23 @@ class Player extends Component {
    *         The current video source when getting
    */
   src(source) {
-    if (source === undefined) {
```

Comment:
I think we should maintain this functionality. Though, having `player.src({})` produce the same error as `player.src([])` sounds fine too.